[docs] Small fixes (#357)
MystenLabs/seal: docs/UsingSeal.md

Note that the encryption does **not** conceal the size of the message. If message size is considered sensitive, pad the message with zeros until its length no longer reveals meaningful information.

!!! note

>     Seal can be used for encrypting an ephemeral symmetric key, which you can use to encrypt your actual content. This approach is useful when storing encrypted data immutably on Walrus while keeping the encrypted key separately on Sui. By managing the key separately, you can update access policies or rotate key servers without modifying the stored content.

Reviewer: nit: again, it was as desired in active voice

});
```

> Seal evaluates the transaction as if it was sent by the user, i.e., `TxContext::sender()` is expected to return the account that signed on the session key.

Reviewer: ```suggestion
Seal evaluates the transaction as if the user sent it. In Move, `TxContext::sender()` returns the account that signed with the session key.
```

To decrypt an encrypted object in a Move package, follow these steps:

- **On-chain app initialization**
    - Retrieve public keys with `client.getPublicKeys` and convert them with `bf_hmac_encryption::new_public_key`.
    - Store the key server public keys on-chain.

>     - The dapp/Users should verify the correctness of those public keys before uploading their encryptions.

Author: i think the other version is clearer

- **Verify derived keys**
    - Use the Seal SDK client to fetch derived keys via `client.getDerivedKeys`, which returns a map of key server object IDs to their derived keys.

>     - Convert from bytes to `Element<G1>` or `Element<G2>` using the [`from_bytes`](https://docs.sui.io/references/framework/sui/group_ops#sui_group_ops_from_bytes) function.

Reviewer: ```suggestion
    - Convert bytes to `Element<G1>` or `Element<G2>` with [`from_bytes`](https://docs.sui.io/references/framework/sui/group_ops#sui_group_ops_from_bytes).
```
